src/test/java/lotto/LottoDomainTest.java

```diff
@@ -0,0 +1,53 @@
+package lotto;
```

Comment:
테스트의 패키지를 프로덕션 패키지와 동일하게 구성해 주시면 좋을 것 같아요
```suggestion
package lottoController;
```

Reply:
테스트의 패키지를 프로덕션 패키지와 동일하게 구성  완료.
- test/java/lottoController
- test/java/lottoModel